kamailio/dispatcher-role.cfg

```diff
@@ -36,13 +36,13 @@ route[DISPATCHER_CLASSIFY_SOURCE]
     if (is_myself("$ou")) {
         xlog("L_INFO", "$ci|log|original R-URI ($ou) is this proxy, treating as external sources\n");
     } else if (
-      ds_is_from_list(1, 3) ||
-      ds_is_from_list(2, 3) ||
-      ds_is_from_list(3, 3) ||
-      ds_is_from_list(10, 3) ||
-      ds_is_from_list(20, 3)
+      ds_is_from_list(1, 3, "$rz:$rd:$rp") ||
+      ds_is_from_list(2, 3, "$rz:$rd:$rp") ||
+      ds_is_from_list(3, 3, "$rz:$rd:$rp") ||
+      ds_is_from_list(10, 3, "$rz:$rd:$rp") ||
+      ds_is_from_list(20, 3, "$rz:$rd:$rp")
```

Comment:
@sergey-safarov what's the intent changing this ?

Reply:
This fix broken hold/unhold feature when enabled TRAFFIC_FILTER_ROLE

Reply:
@sergey-safarov then maybe the problem is with the TRAFFIC_FILTER_ROLE ? i feel this would fail when the ruri is the device but source ip/port is freeswitch. we should look at traffic_filter_role first before trying to change here since grandstream's work without traffic_filter_role